src/test/kotlin/org/rust/ide/refactoring/RsExtractFunctionTest.kt

```diff
@@ -1176,6 +1176,68 @@ class RsExtractFunctionTest : RsTestBase() {
         false,
         "foo")
 
+    fun `test import parameter types`() = doTest("""
+        use a::{S, foo};
+
+        mod a {
+            pub struct A;
+            pub fn foo() -> A { unimplemented!() }
+        }
+
+        fn main() {
+            let s = foo();
+            <selection>s;</selection>
+        }
+    """, """
+        use a::{S, foo, A};
+
+        mod a {
+            pub struct A;
+            pub fn foo() -> A { unimplemented!() }
+        }
+
+        fn main() {
+            let s = foo();
+            bar(s);
+        }
+
+        fn bar(s: A) {
+            s;
+        }
+    """,
+        false,
```

Comment:
[minor] Looks like `doTest` could have the following signature 
```Kotlin
private fun doTest(
    @Language("Rust") code: String,
    @Language("Rust") excepted: String,
    name: String,
    pub: Boolean = false,
    noSelected: List<String> = emptyList(),
    mutabilityOverride: Map<String, Boolean> = emptyMap()
)
```
since we use `pub = true` only twice